kong/plugins/jwt/handler.lua

```diff
@@ -47,23 +48,24 @@ local function retrieve_token(conf)
       if type(token_header) == "table" then
         token_header = token_header[1]
       end
-      local iterator, iter_err = re_gmatch(token_header, "\\s*[Bb]earer\\s+(.+)")
+      local iterator, iter_err = re_gmatch(token_header, ",?\\s*[Bb]earer\\s+([\\w\\d+.=-]+)\\s*,?")
```

Comment:
I do not think this is necessary to have multiple tokens in one `Authorization` header. You can have multiple `Authorization` headers. So plugin here should pass all of them through.

Reply:
That’s actually not true

>    A sender MUST NOT generate multiple header fields with the same field
   name in a message unless either the entire field value for that
   header field is defined as a comma-separated list [i.e., #(values)]
   or the header field is a well-known exception (as noted below).

Set-Cookie is the exception. Auth is csv. 

[reference](https://tools.ietf.org/html/rfc7230#section-3.2.2)